indico/modules/events/surveys/controllers/management/survey.py

```diff
@@ -163,3 +169,68 @@ def _send_emails(self, form, recipients):
             email = make_email(to_list=recipient, bcc_list=bcc, from_address=form.from_address.data,
                                template=tpl, html=True)
             send_email(email, self.event, 'Surveys')
+
+
+class RHEmailEventSurveyBase(RHManageSurveyBase):
+    """Sends emails to event participants."""
```

Comment:
```suggestion
    """Send emails to event participants."""
```

docstrings should use imperative wording